Source/diablo.cpp

```diff
@@ -322,6 +326,7 @@ static void run_game_loop(unsigned int uMsg)
 		if (!gbRunGame)
 			break;
 		if (!nthread_has_500ms_passed()) {
+			plrctrls_every_frame();

```

Comment:
Why not call this from ProcessInput or FetchMessage?

Reply:
Done